maptivist/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -107,13 +108,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         PreparedQuery results = datastore.prepare(query);
  
         for (Entity entity : results.asIterable()) {
-            String title = (String) entity.getProperty("title");
-            String description = (String) entity.getProperty("description");
-            Double longitude = Double.parseDouble((String) entity.getProperty("longitude"));
-            Double latitude = Double.parseDouble((String) entity.getProperty("latitude"));
-            Set<String> links = createLinkObject((String) entity.getProperty("links"));
+            String title = (String) entity.getProperty("marker-title");
+            String description = (String) entity.getProperty("marker-description");
+            Double longitude = Double.parseDouble((String) entity.getProperty("marker-lng"));
+            Double latitude = Double.parseDouble((String) entity.getProperty("marker-lat"));
+            Set<String> links = createLinkObject((String) entity.getProperty("marker-link"));
             ArrayList<String> flags = createFlagObject((String) entity.getProperty("flags"));
-            Set<String> categories = createCategoriesObject((String) entity.getProperty("category"));
+            Set<String> categories = createCategoriesObject((String) entity.getProperty("marker-category"));
```

Comment:
On Line 77-Line 84 above, you store them without the "marker-" prefix, but then you fetch with with the  marker prefix? Is this a bug?